administrator/components/com_content/helpers/content.php

```diff
@@ -39,6 +39,10 @@ public static function addSubmenu($vName)
 			'index.php?option=com_categories&extension=com_content',
 			$vName == 'categories');
 		JHtmlSidebar::addEntry(
+			JText::_('COM_CONTENT_SUBMENU_SHARE'),
+			'index.php?option=com_content&view=share',
+			$vName == 'share');
+		JHtmlSidebar::addEntry(
```

Comment:
Part 1 of shared draft manager- added option shared drafts in submenu of article view